source/features/extend-diff-expander.css

```diff
@@ -5,8 +5,17 @@
 		.blob-num:not(:hover) .directional-expander:first-child,
 		.blob-num:not(:hover) + .blob-code
 	) {
-	color: var(--color-state-hover-primary-text, #fff);
-	background: var(--color-state-hover-primary-bg, #0366d6);
-	border-color: var(--color-state-hover-primary-border, #0366d6);
+	color: var(
+		--control-checked-fgColor-rest,
```

Comment:
<img width="235" alt="Screenshot 1" src="https://github.com/refined-github/refined-github/assets/1402241/229e09e2-4210-4dcb-b2c3-a04d42400902">

Some of the colors are not right, this should match the color it extends. It would be great if you could verify each color in the dark theme via screenshots 🥹 🙏 

Here the right one `var(--bgColor-accent-emphasis, var(--color-accent-emphasis))`

![Uploading Screenshot 2.png…]()

Reply:
> unless it's an exact match

Here are the files that don't have exact matches:

<img width="608" alt="Screenshot 3" src="https://github.com/refined-github/refined-github/assets/1402241/276f5f0e-1157-42a8-94ce-c3dffe839f29">

Reply:
This is still not resolved, the background should not be white

Reply:
Oh yeah it works 🤔 

<img width="312" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/64000a38-96ed-4e25-a138-175d2ab49847">